substratevm/src/com.oracle.svm.core.graal.llvm/src/com/oracle/svm/core/graal/llvm/NodeLLVMBuilder.java

```diff
@@ -796,6 +797,12 @@ public Value setResult(ValueNode node, Value operand) {
                         "value type doesn't match node stamp (" + node.stamp(NodeView.DEFAULT).toString() + ")", llvmOperand.get());
 
         gen.getDebugInfoPrinter().setValueName(llvmOperand, node);
+        if (LLVMOptions.IncludeLLVMSourceDebugInfo.getValue()) {
+            if (llvmOperand.get() instanceof LLVMValueRef) {
```

Comment:
As far as I can tell `llvmOperand.get()` is always an `LLVMValueRef`

Reply:
Agreed, I'll remove this check